### DISCUSSION TITLE
EPG data inconsistent

### DISCUSSION THREAD

**Discussion by xjim1** [OpenSageTV/Community#145] [Guide/EPG]
Has anyone else been experiencing incorrect information in the programming guide?  I've seen an increase in season and episode descriptions that don't match at all what was recorded.  I only record OTA channels, so I don't know if that matters.  I see it on multiple channels.  One channel only displays basic information(no season/episode info at all) and for only 12hrs ahead of time.  

My conversations with SD point the finger at the local stations, but too many channels are showing this symptom.  I don't see this as a Sage problem as much as potential nails in the SD coffin.
Any thoughts?

**Comment by nyplayer** [comment]
SD gets guide data from GraceNote ... Crap in Crap out so not SD fault.

**Comment by emveepee** [comment]
It's not just Gracenote, check "House of the Dragon" on TV Maze, IMDB TMDB etc.  Tonight's and future episode don't have a subtitle or episode specific description yet.  After tonight I expect Episode 6 will get populated but by then recordings will be saved with the stale data.  It probably boils down to when and how this information is publicly released.  It isn't even on the HBO Canada web site right now.

Many shows are like this, be thankful that S/E are correct.

**Comment by xjim1** [comment]
I'm not familiar where those sites get their data, but aren't some of them user-contributed? I know very little how the metadata 'supply-chain' works, but I assume for broadcast channels, it is provided to the syndicated(local) station by the content providers(CBS, PBS, MeTV, H&I, etc), then out to a distributor. 

For some first-run episodes, I'll see it updated the day of or after being recorded, but the problems I have are with re-runs.  The S/E and descriptions are of other episodes and never get updated.  

In the past, I've compared to what Zap2it or TitanTV showed before I've logged a ticket with SD.  Nowadays, I'll get different info from each one of them, though TitanTV is the most accurate.  And PBS, for example, shows correct episode info on their website, so either it's not being sent out or getting dropped somewhere.  Trying to communicate with these stations is like talking to a rock.

Anyway, still like to see if anyone's seeing this level of disparity.
Cheers

**Reply by emveepee** [reply]
Zap2It is TMS (Gracenote) and TitanTV uses Gracenote.  Silicondust and Emby also use Gracenote like SD so it is probably just all their backend synced up.  Best to compare with TV Passport if you want different data.
